docs/streams/index.html

```diff
@@ -237,67 +237,59 @@ <h3 style="margin-top: 5.3rem;">Hello Kafka Streams</h3>
-                   import java.lang.Long
-                   import java.util.Properties
-                   import java.util.concurrent.TimeUnit
-
-                   import org.apache.kafka.common.serialization._
-                   import org.apache.kafka.common.utils.Bytes
-                   import org.apache.kafka.streams._
-                   import org.apache.kafka.streams.kstream.{KStream, KTable, Materialized, Produced}
-                   import org.apache.kafka.streams.state.KeyValueStore
-
-                   import scala.collection.JavaConverters.asJavaIterableConverter
-
-                   object WordCountApplication {
-
-                       def main(args: Array[String]) {
-                           val config: Properties = {
-                               val p = new Properties()
-                               p.put(StreamsConfig.APPLICATION_ID_CONFIG, "wordcount-application")
-                               p.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "kafka-broker1:9092")
-                               p.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass)
-                               p.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass)
-                               p
-                           }
-
-                           val builder: StreamsBuilder = new StreamsBuilder()
-                           val textLines: KStream[String, String] = builder.stream("TextLinesTopic")
-                           val wordCounts: KTable[String, Long] = textLines
-                               .flatMapValues(textLine => textLine.toLowerCase.split("\\W+").toIterable.asJava)
-                               .groupBy((_, word) => word)
-                               .count(Materialized.as("counts-store").asInstanceOf[Materialized[String, Long, KeyValueStore[Bytes, Array[Byte]]]])
-                           wordCounts.toStream().to("WordsWithCountsTopic", Produced.`with`(Serdes.String(), Serdes.Long()))
-
-                           val streams: KafkaStreams = new KafkaStreams(builder.build(), config)
-                           streams.start()
-
-                           Runtime.getRuntime.addShutdownHook(new Thread(() => {
-                               streams.close(10, TimeUnit.SECONDS)
-                           }))
-                       }
+import java.util.Properties
+import java.util.concurrent.TimeUnit
 
-                   }
+import org.apache.kafka.streams.kstream.Materialized
+import org.apache.kafka.streams.scala.kstream._
+import org.apache.kafka.streams.{KafkaStreams, StreamsConfig}
+
+object WordCountApplication extends App {
+  import DefaultSerdes._
+  import ImplicitConversions._
+
+  val config: Properties = {
+    val p = new Properties()
+    p.put(StreamsConfig.APPLICATION_ID_CONFIG, "wordcount-application")
+    p.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "kafka-broker1:9092")
+    p
+  }
+
+  val builder: StreamsBuilder = new StreamsBuilder()
```

Comment:
This example code doesn't compile, because e.g. the import for `StreamsBuilder` is missing (from package `org.apache.kafka.streams.scala`).

I would probably double-check the other examples that are shown in the documentation, too.

Reply:
@seglo could you take a look?